scripts/fn_paradrop.sqf

```
@@ -0,0 +1,216 @@
+ * increasing flyInHeight before using it.  Also, don't try with helicopters.
+ *
+ * Optional Arguments: Reset Height
+ * For drop purposes flyInHeight is set during the run.  If the transport is to be
+ * otherwise used after the run, set this to <true> in order to resume normal behavior
+ *
+ * Optional Arguments: Disable Waypoint 2
+ * Occasionally, AI behavior can be wonky especially when drop WPs are around airports.
+ * This will cause them to veer off from their drop direction and attempt to land. 
+ * Disabling WP 2 stops this behavior from occurring on the second waypoint.
+ * If it occurs on the first, I recommend changing the DZ area, the drop length,
+ * or the approach direction. 
+ *
+ * Arguments
+ * 0: Pilot Group <GROUP>
+ * 1: Drop Groups <ARRAY>
+ * 2: Drop Target <MARKER>
+ * 3: Drop Mode <ARRAY> or <NUMBER> (default: 0)
+ * 4: Reset Height <BOOL> (default: false) 
+ * 5: Disable Waypoint 2 <BOOL> (default: false)
+ *
+ * Return Value:
+ * None
+ *
+ * Examples:
+ * [pilot_group, [airborne_1], "drop_marker"] call TAC_Scripts_fnc_paradrop;
+ * [pilot_group, [airborne_1,airborne_2], "drop_marker"] call TAC_Scripts_fnc_paradrop;
+ * [pilot_group, [airborne_1], "drop_marker", 1] call TAC_Scripts_fnc_paradrop;
+ * [pilot_group, [airborne_1], "drop_marker", 0, false, false] call TAC_Scripts_fnc_paradrop;
+ * [pilot_group, [airborne_1], "drop_marker", [200,1000,150]] call TAC_Scripts_fnc_paradrop;
+ */
+
+#define DROP_MODES [[150, 1000, 140], [105, 1000, 80], [250, 1000, 140] ,[4500, 1000, 900], [3000, 1000, 900]]
+
+params ["_pilotGroup", "_dropGroups", "_loc", ["_dropMode", 0], ["_resetHeight", false], ["_disableWP2", false]];
+
+if (_loc isEqualTo "") exitWith {ERROR("Paradrop given invalid marker!")};
+
+private _dropData = [];
+// first process dropMode
+if (_dropMode isEqualType 0) then {
+    if (_dropMode < 0 || {_dropMode > 4}) then {
+        ERROR("Paradrop mode %1 invalid - defaulting to 0!",_dropMode);
```

Why did you change this? I wrote it correctly, or it won't work at all (silent fail due to macros error handling).
```suggestion
        ERROR_1("Paradrop mode %1 invalid - defaulting to 0!",_dropMode);
```